File: Source/Microsoft.Teams.Apps.EmployeeTraining/ClientApp/src/models/event-operation-type.ts
// <copyright file="event-operation-type.ts" company="Microsoft">
// Copyright (c) Microsoft. All rights reserved.
// </copyright>

/** Indicates the event operation */
export enum EventOperationType {
    /** No event operation */
    None,

    /** The operation in task module to close event registrations */
    CloseRegistration,

    /** The operation in task module to cancel an event to occur */
    CancelEvent,

    /** The operation in task module to register for an event */
    Register,

    /** The operation in task module to cancel event registration */
    Remove
}

File: Source/Microsoft.Teams.Apps.EmployeeTraining/Models/Enums/EventOperationType.cs
// <copyright file="EventOperationType.cs" company="Microsoft">
// Copyright (c) Microsoft. All rights reserved.
// </copyright>

namespace Microsoft.Teams.Apps.EmployeeTraining.Models.Enums
{
    /// <summary>
    /// Indicates the event operation
    /// </summary>
    public enum EventOperationType
    {
        /// <summary>
        /// No event operation
        /// </summary>
        None,

        /// <summary>
        /// The close event registration
        /// </summary>
        CloseRegistration,

        /// <summary>
        /// Cancel an event
        /// </summary>
        CancelEvent,
    }
}
